kickstarts/partials/post/source_setup.ks.erb

```diff
@@ -9,22 +9,25 @@ ssui_root="$repos_root/manageiq-ui-self_service"
 
 rm -rf $app_root
 
-git clone <%= @appliance_checkout.remote %> $appliance_root
+git clone <%= "--separate-git-dir /tmp/appliance.git" if @build_type == "release" %> \
+        <%= @appliance_checkout.remote %> $appliance_root
```

Comment:
As [discussed above](https://github.com/ManageIQ/manageiq-appliance-build/pull/149#discussion-diff-75916773) we were thinking that nightlies would have the .git directory, but release builds would not.  Does that sound ok to you or do you think that would cause a problem.